### DISCUSSION TITLE
Syncronize Directories - Empty Directories

### DISCUSSION THREAD

**Discussion by VictorTrifu** [doublecmd/doublecmd#1857] [Suggestions]
### Version

Double Commander
Version: 1.1.18 gamma
Revision: 568
Commit: c45365d1b
Build date: 2024/09/21
Lazarus: 3.5.0.0
FPC: 3.2.2
Platform: x86_64-Win64-win32/win64
OS version: Windows 10 Home 22H2 x86_64

### Description

Thank you again for creating the Double Commander and fixing the bugs so quickly.

This is basically not a bug but it feels like one. Feel free to move it to the feature request if you want.

In Total Commander you have in synchronized directories a check box called "empty directories". When I use the asymmetric option to backup my data with the "empty directory" checkbox checked, TC would delete the empty directories on my backup drive if I have deleted  or renamed that directory on my source drive.

With Double Commander I have to manually go over each directory look for the empty ones and delete them on my backup drive which is a lot of an unnecessary work. The whole point of the asymmetric backup is that you have a copy of the source on the backup drive.  Without this option this is not really possible. Please consider adding this option to the synchronized directories in Double Commander so that a proper backup can be realized. 

Please let me know if I missing something.

Thank you very much in advance. DC is very cool!

**Comment by alexx2000** [comment]
#1163
